src/main/flight/pid.h

```diff
@@ -324,6 +330,17 @@ typedef struct pidProfile_s {
     uint8_t yaw_type;                   // For wings: type of yaw (rudder or differential thrust)
     int16_t angle_pitch_offset;         // For wings: pitch offset for angle modes; in decidegrees; positive values tilting the wing down
 
+    // use aerodynamics
+    uint8_t  ad_mode;                   //For wings: The aerodynamics using mode: off, angle of attack, tpa
```

Comment:
IMO this should go into separate PG, at least after testing. Is there any advantage to select configuration during flight?